ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -53,15 +132,33 @@ export default function EditGasDisplay({
             </Typography>
           </div>
         )}
-        <TransactionTotalBanner total="" detail="" timing="" />
+        <TransactionTotalBanner total={maxFeeFiat} detail="" timing="" />
```

Comment:
as I review the how to use doc, I think this number the big one is supposed to use '(estimatedBaseFee + maxPriorityFeePerGas) * gasLimit'

> The big number should be `(estimatedBaseFee + (customMaxPriorityFeePerGas || selectedFeeEstimate.suggestedMaxPriorityFeePerGas)) * gasLimit` and then converted to fiat